### PR TITLE
Account buttons cleanup

### PR DESCRIPTION
**Pull request by zarvox**
* Fix bug where identity cards weren't looking up the current identity correctly
* Give identity cards some margin so they don't look weird when stacked
* Audit the codebase for use of `{{> someTemplate }}` and convert to either `{{> someTemplate . }}` (explicitly passing on data context) or `{{> someTemplate ""}}` (passing on an empty data context).
* Add a couple typechecks on the args to the login/account button templates

### CODE REVIEW

**Comment by dwrensha** [comment]
Could you upload a screenshot of the new spacing?

**Comment by zarvox** [comment]
I appear to have some bugs to fix, but:

spaced-identity-choices

**Comment by zarvox** [comment]
Okay, this is in better shape now.

**Comment by neynah** [comment]
The pure black border looks harsh to me. I think we should be consistent and use something closer to #b3b3b3. The spacing looks good though.

**Comment by kentonv** [comment]
Personally I find `Template.currentData()` confusing because at a glance I tend to interpret it as `Template.instance().data`, because in my mind the current data context would have been accessed as `this` and not `Template.something`.

**Comment by zarvox** [comment]
Yeah, I could see that.

I find it kind of annoying that `this` is the template instance in `Template.onCreated()` and `Template.onRendered()`, and if you want to reactively use the data context, then you need to use `Template.currentData()`, whereas in template helpers and event maps `this` is the data context, and you have to use `Template.instance()` to get a handle to the template instance.

I guess I'll try to leave these alone in future patchsets, then?

**Comment by kentonv** [comment]
Hmm, in `onCreated()` you can use `this.data`, right?

But yeah, the inconsistency is annoying.

Anyway, no action needed at this time.

File: shell/client/accounts/account-settings.html [hunk 1]
```diff
@@ -229,7 +229,7 @@ <h3 class="title-bar">Primary e-mail address for service-related notifications</
             </li>
           {{/with}}
           {{#if isPaymentsEnabled}}
-            {{> billingOptins}}
+            {{> billingOptins "" }}
```
Comment (kentonv) [inline shell/client/accounts/account-settings.html]: This broke the mailing list signup prompt, which needs `db` in order to display the user's current storage quota.

I've fixed this, but I'm worried that it's hard to know how many other things might be broken.

**Pull request merged**